views/learn.ejs

```diff
@@ -38,34 +38,76 @@
+                to toggle between the original vulnerable class (security rating 0), and the newly secured class (security rating 1).
+                We also added some security features not found in the original DVNA, such as (i) enforcing users to choose stronger passwords; and (ii) ability to
+                choose different password encryptions and salt them.  
+                <br><br>
+                We documented all our findings and work in our  <a href="https://app.gitbook.com/@dvna-dep/s/dvna-dep/" target="_blank" rel="noopener noreferrer">DVNA-DEP security guide. 
+                </a>.  The security guide, which covers classes A1 to A6 and password hash/salt, is also a major part of our project.
+                It delves into those security issues much deeper than the <a href='https://appsecco.com/books/dvna-developers-security-guide/' target="_blank" rel="noopener noreferrer">original Appsecco security guide</a>.  Further, it contains information on
+                installing and using the DVNA-DEP app.    
+                <Br><br>
+
+                <div class="container-fluid">
+
+                <div class="row">
+                    <div class="col-md-4">
+                        <h4>OWASP Top 10 2017</h4>
+                        <h6>Note: DVNA-DEP only expanded A1 to A6 sections below. The rest work exactly the same as the original DVNA</h6>
+                        <ul>
+                            
+                            <% for (var vulnKey in vulnerabilities) { %>
+                                <% if (vulnKey[1]!='x') { %>
+                                <li><a href='/learn/vulnerability/<%=vulnKey%>'> <%=vulnerabilities[vulnKey]%> </a></li>
+                                <% } %>
+                            <% } %>
+                        </ul>               
+                    </div>
+                    <div class="col-md-4">
+                        <h4>OWASP Top 10 2013</h4>
+                        <h6>Note: DVNA-DEP did not expand below sections. They work exactly the same as the original DVNA.</h6>
+                        <ul>
+                            <% for (var vulnKey in vulnerabilities) { %>
+                                <% if (vulnKey[1]=='x') { %>
+                                <li><a href='/learn/vulnerability/<%=vulnKey%>'> <%=vulnerabilities[vulnKey]%> </a></li>
+                                <% } %>
+                            <% } %>
+                        </ul>       
+                    </div>
+                    <div class="col-md-4">
+                        <h4>External Links</h4>
+                        <h5>DVNA-DEP</h5>
+                        <ul>
+                            <li><a href='https://github.com/dvna-dep/dvna-dep' target="_blank" rel="noopener noreferrer">DVNA-DEP on github</a></li>
+                            <li><a href='https://app.gitbook.com/@dvna-dep/s/dvna-dep/' target="_blank" rel="noopener noreferrer">DVNA-DEP Security Guide</a></li>
```

Comment:
I think this is the link that points to the app. It redirected me properly to the static site where the book is hosted, but we probably want to point to that directly. Suggest pointing the link to the static site. 
```suggestion
                            <li><a href='https://dvna-dep.gitbook.io/dvna-dep/' target="_blank" rel="noopener noreferrer">DVNA-DEP Security Guide</a></li>
```